central/role/service/service_impl.go

```diff
@@ -117,12 +117,6 @@ func (s *serviceImpl) CreateRole(ctx context.Context, roleRequest *v1.CreateRole
 	}
 	role.Name = roleRequest.GetName()
 
-	// Empty access scope ID is deprecated. Fill the default during the adoption
-	// period.
-	// TODO(ROX-9510): remove this block.
```

Comment:
This only partially resolves ROX-9510. Let's either address ROX-9510 completely or leave this out. I'm fine with any option.

Reply:
Will revert this change and defer the changes to Authz team.